frostsnap_core/src/coordinator.rs

```diff
@@ -483,6 +483,7 @@ impl FrostCoordinator {
 
     pub fn start_sign(
         &mut self,
+        key_id: KeyId,
```

Comment:
These KeyId changes make sense to me, but I am wondering about how far the scope of frostsnap_core should extend.

Is it actually important for Frostsnap core to know about various keys? Though I can't think of a better place to handle it currently. Maybe you've thought about this with upcoming multiple keys PR

Reply:
It's a good point. It could be removed for now. It got added in place of something more complicated I was doing which was making the sign task have the key in it.